src/decision/dMCDA.jl

```diff
@@ -360,12 +360,18 @@ function create_decision_matrix(
 
     # Wave damage, account for cases where no chance of damage or heat stress
     # if max > 0 then use damage probability from wave exposure
-    A[:, 4] .= maximum(wave_stress) != 0.0 ? (wave_stress .- minimum(wave_stress)) ./ (maximum(wave_stress) - minimum(wave_stress)) : 0.0
-    #A[:, 4] .= wave_stress ./ maximum(wave_stress)
+    A[:, 4] .= if maximum(wave_stress) != 0.0
+        (wave_stress .- minimum(wave_stress)) ./ (maximum(wave_stress) - minimum(wave_stress))
+    else
+        0.0
+    end
 
     # risk from heat exposure
-    # A[:, 5] .= maximum(heat_stress) != 0.0 ? (heat_stress .- minimum(heat_stress)) ./ (maximum(heat_stress) - minimum(heat_stress)) : 0.0
-    A[:, 5] .= min.(heat_stress ./ 20.0, 1.0)
+    A[:, 5] .= if maximum(heat_stress) != 0.0
```

Comment:
Same thing as above: feel free to change only if you feel like it:

```
    max_hs = maximum(heat_stress)
    min_hs = minimum(heat_stress)
    A[:, 4] .= 0.0 ? max_hs == 0.0 : (heat_stress .- min_hs) ./ (max_hs - min_hs)
```

Reply:
Agree, I just changed it to be more readable. Eventually I think these will be moved into functionsa but for now this looks better :)